server/src/server.ts

```diff
@@ -95,19 +95,26 @@ let findBinaryDirPathFromProjectRoot = (
     return null;
   }
 
-  return findBinaryDirPathFromProjectRoot(parentDir);
+  return findBinaryPathFromProjectRoot(parentDir);
 };
 
-let getBinaryDirPath = (projectRootPath: p.DocumentUri) =>
-  extensionConfiguration.binaryPath == null
-    ? findBinaryDirPathFromProjectRoot(projectRootPath)
-    : extensionConfiguration.binaryPath;
-
 let findRescriptBinary = (projectRootPath: p.DocumentUri) =>
-  utils.findRescriptBinary(getBinaryDirPath(projectRootPath));
-
-let findBscBinary = (projectRootPath: p.DocumentUri) =>
-  utils.findBscBinary(getBinaryDirPath(projectRootPath));
+  extensionConfiguration.binaryPath == null
+    ? findBinaryPathFromProjectRoot(projectRootPath)
+    : utils.findRescriptBinary(extensionConfiguration.binaryPath);
+
+let findBscBinary = (projectRootPath: p.DocumentUri) => {
+  let rescriptBinaryPath = findRescriptBinary(projectRootPath);
+  if (rescriptBinaryPath !== null) {
+    return path.join(
+      path.dirname(rescriptBinaryPath),
+      "..",
```

Comment:
Is this OK for windows, symbolic links etc? Or should one manipulate the path instead?

Reply:
`..` definitely works on windows